smoke.sh

```diff
@@ -62,10 +63,7 @@ function finish ()
 {
     RET=$?
     if [ $RET -ne 0 ]; then
-        mkdir -p /d/logs/smoke
-        filename="/d/logs/smoke/glusterfs-logs-`date +%Y%m%d%H%M%S`.tgz"
-        tar -czf $filename /build/install/var/log;
-        echo Logs archived in $filename
```

Comment:
There's nothing in `/build/install/var/log;` on any machine. So we've most likely gotten the path wrong. Right now it's not doing anything useful (except maybe throwing a false flag)